user_settings.sql

```diff
@@ -0,0 +1,94 @@
+-- This is just a test (in draft) file so that Arit and I can work together using revision control
+-- We run the contents of this file in the rails dbconsole
+-- It will not be committed to the codebase, it will be moved into a data update script and this file will be deleted :)
+
+BEGIN TRANSACTION;
+WITH settings_data AS (
+  SELECT
+    users.id AS user_id,
+    CASE WHEN config_font='default' THEN 0
```

Comment:
**nitpick, non-blocking:** can we alphabetize the column names just to make it easier to read and find the names?

Reply:
Yes will do 👍🏾